src/main/webapp/style.css

```diff
@@ -319,24 +326,58 @@ img {
   color: red;
 }
 
-#create-route-button {
+#route-content button {
   justify-content: center;
   align-items: center;
   margin-top: 15px;
   margin-bottom: 5px;
   height: 25px;
-  width: 120px;
+  width: auto;
   border: solid rgb(163, 163, 163) 1px;
   background-color: rgb(255, 255, 255, 0.8);
   color: rgb(116, 116, 116);
   border-radius: 7px;
   cursor: pointer;
 }
 
-#create-route-button:hover {
+#route-content button:hover {
   color: darkgray;
 }
 
+#share-section {
+  visibility: hidden;
+  position: absolute;
+  width: 200px;
+  height: 100px;
+
+  border: solid rgb(163, 163, 163) 1px;
+  background-color: rgb(255, 255, 255, 0.8);
+  border-radius: 7px;
+
+  right: 0;
+  left: 0;
+  margin-left: auto;
+  margin-right: auto;
+  bottom: 35px;
+}
+
+#share-section.show {
+  visibility: visible;
+  -webkit-animation: fadeIn 1s;
+  animation: fadeIn 1s;
+}
+
+#share-section input {
+  height: 25px;
```

Comment:
Can you please also add a `width` value for the input, less than 200px, because it is a little out of its section on my preview.